docs/_pages/web_client.md

```diff
@@ -233,164 +231,6 @@ Note that when calling a method with no required arguments, you **still need to
 
 ---
 
-### Using refresh tokens
```

Comment:
I think removing this section (and the following section) would leave users who have already built a Workspace App without information about how these options work. Instead, I think we can add warnings (similar to how the reference documentation on the options to the constructor were given warnings) to help users understand that this usage only works with the now-deprecated workspace app tokens (xoxa).

Reply:
Sounds good!